x-pack/legacy/plugins/siem/public/pages/hosts/index.tsx

```diff
@@ -95,7 +92,6 @@ export const HostsContainer = pure<HostComponentProps>(({ match }) => (
                     to={to}
                     setQuery={setQuery}
                     isInitializing={isInitializing}
-                    {...props}
```

Comment:
Note: I removed props spreading as that was causing additional properties such as a `match` and `location` being sent down into the component which causes additional rendering.

Props spreading is interesting in that TypeScript does not alert us to the additional un-needed attributes being pushed down. Any extra values being pushed down can cause additional rendering such as in our case where `match` will be a new object and cause re-rendering.